RedHat-UX/red-hat-design-system
# docs(stat): statistic docs cleanup

## What I did

1. Added the stat content docs for `rh-stat` from #931 

## Testing Instructions

1. Please go to the Deployment Preview
2. Please check the outline / xd / content docs and ensure the content matches:
  - `Overview`
  - `Style`
  - `Guidelines`
  - `Code`
  - `Accessibility`
